SEED-platform/seed
# fix duplicate column errors

#### Any background context you want to provide?
The SEED analysis functionality is able to create new columns as needed. In some cases, the `column_name` may already exist in the database and it will create a new instance of the column with the same `column_name` but different `display_name` or `column_description`. This was causing duplicate column errors in the database.

#### What's this PR do?
1. creates a migration to remove certain duplicate fields. It only checks for mapped fields (where `table_name` is not None). If there is more than one of the fields, it removes the most recent. This may cause data loss if the user ran an analysis that used this field after the GHG fields were updated. Presently, there are only a few orgs (and they are all test orgs) that are experiencing this issue. The migration should handle any mappings.
2. Update the Column.objects.get_or_create method on the analyses to first `get_or_create` on the minimum "non-user" configurable data. After finding or creating the column, then assign the `column_display_name` and `column_description`. It only assigns the fields if it is created since the user may have renamed the display name or column description. 

#### How should this be manually tested?
* Run the migrations on your local DB and make sure that everything works.
* For bonus points, download the latest DB dump and run on the production, staging, or dev1.

#### What are the relevant tickets?
#3325 

#### Screenshots (if appropriate)
